tower-http/CHANGELOG.md

```diff
@@ -7,7 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 # Unreleased
 
-- None.
+- `ServeDir` and `ServeFile`: Ability to serve precompressed files ([#156])
```

Comment:
I think its missing a link to `[#156]` 🤔

Reply:
Huh? Isn't that the last part `([#156])` or do I need to do something else as well?

Reply:
You need to add link for it like its done for the other changelog entries. GH doesn't pick it up automatically unfortunately.

It'll look like this otherwise:

![image](https://user-images.githubusercontent.com/718941/141769334-64cb087d-78ef-4a00-a6c5-2d05845e4549.png)

Reply:
ah right completely missed that, fixed now!